frontend/talentsearch/src/js/components/applications/ApplicationActions.tsx

```diff
@@ -0,0 +1,305 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { Button, Link } from "@common/components";
+import { getLocale } from "@common/helpers/localize";
+import { notEmpty } from "@common/helpers/util";
+
+import AlertDialog from "@common/components/AlertDialog";
+import { toast } from "react-toastify";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+
+import type { Application } from "./ApplicationCard";
+import {
+  useArchiveApplicationMutation,
+  useDeleteApplicationMutation,
+} from "../../api/generated";
+import getFullPoolAdvertisementTitle from "../pool/getFullPoolAdvertisementTitle";
+
+interface ActionProps {
+  show: boolean;
+}
+
+interface ContinueActionProps extends ActionProps {
+  application: Application;
+}
+
+const ContinueAction = ({ show, application }: ContinueActionProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useDirectIntakeRoutes();
+  const { poolAdvertisement } = application;
+
+  if (!show) {
+    return null;
+  }
+
+  return (
+    <Link href={paths.poolApply(application.id)}>
+      {intl.formatMessage(
+        {
+          defaultMessage: "Continue my application<hidden> {name}</hidden>",
+          description: "Link text to continue a specific application",
+        },
+        {
+          name: notEmpty(poolAdvertisement?.name)
+            ? poolAdvertisement?.name[locale]
+            : application.id,
+        },
+      )}
+    </Link>
+  );
+};
+
+interface SeeAdvertisementActionProps extends ActionProps {
+  advertisement: Application["poolAdvertisement"];
+}
+
+const SeeAdvertisementAction = ({
+  show,
+  advertisement,
+}: SeeAdvertisementActionProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useDirectIntakeRoutes();
+
+  if (!show || !advertisement) {
+    return null;
+  }
+
+  return (
+    <Link href={paths.poolAdvertisement(advertisement.id)}>
```

Comment:
This link always fails for me with a "Error, pool unable to be loaded" error message. The regular pool page without the /view works though.  Any idea why?  Maybe it's just my environment.

Reply:
Hmm, I can't seem to reproduce this one. @vd1992 Since you are already reviewing, can you take a look at this as well?

Reply:
this is clicking "See Advertisement" on the card right? works for me
although, it appears that page could do with some styling, but that is definitely out of scope

![image](https://user-images.githubusercontent.com/40485260/188212816-517e5b5a-d3af-49c7-b1de-d8720fafd251.png)

Reply:
Yeah, it does seem to have taken a hit after the hydrogen upgrade. I created an issue for that https://github.com/GCTC-NTGC/gc-digital-talent/issues/3829

Reply:
Oh, my pool wasn't published so it couldn't be seen by an applicant.  nvm